models/ssh_key.go

```diff
@@ -104,8 +105,10 @@ func extractTypeFromBase64Key(key string) (string, error) {
 
 // parseKeyString parses any key string in OpenSSH or SSH2 format to clean OpenSSH string (RFC4253).
 func parseKeyString(content string) (string, error) {
-	// Transform all legal line endings to a single "\n".
-	content = strings.NewReplacer("\r\n", "\n", "\r", "\n").Replace(content)
+	// Transform all legal line endings to a single "\n"
+	newlineExpression := regexp.MustCompile(`\r?\n`) // windows style optional with (\r?)
+	content = newlineExpression.ReplaceAllString(content, "\n")
```

Comment:
That is ture, but we should not get \r only, if that happens it means the input end is not a user with an interface ... it is a script.